<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>435</width>
    <height>266</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Remote Mouse Server</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <property name="sizePolicy">
    <sizepolicy hsizetype="Maximum" vsizetype="Maximum">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_2">
    <item>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <widget class="QPushButton" name="button_start">
        <property name="text">
         <string>Start Server</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="button_stop">
        <property name="text">
         <string>Stop Server</string>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="verticalSpacer">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>40</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QPushButton" name="editClientsButton">
        <property name="text">
         <string>Edit Clients</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <widget class="QTableView" name="table_clients">
      <property name="cornerButtonEnabled">
       <bool>false</bool>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections>
  <connection>
   <sender>button_start</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>startServer()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>36</x>
     <y>20</y>
    </hint>
    <hint type="destinationlabel">
     <x>39</x>
     <y>71</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>button_stop</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>stopServer()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>49</x>
     <y>50</y>
    </hint>
    <hint type="destinationlabel">
     <x>53</x>
     <y>89</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>editClientsButton</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>editIds()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>37</x>
     <y>224</y>
    </hint>
    <hint type="destinationlabel">
     <x>52</x>
     <y>182</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>startServer()</slot>
  <slot>stopServer()</slot>
  <slot>editIds()</slot>
 </slots>
</ui>
